migrations/20201121211100-update negative runNumbers to null.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.bulkUpdate(
+      'patients',
+      { runNumber: null },
+      { runNumber: '-1' }
+    );
+  },
+  down: () => {
+    return Promise.resolve();
+  },
```

Comment:
nit: can save 2 lines 🧑‍🔬 
```suggestion
  down: () => Promise.resolve(),
```